.env

```diff
@@ -0,0 +1,2 @@
+MONGODB_URL=mongodb+srv://mongoadmin:p2KmBxZFi4KB4F51@cluster0.mayja.mongodb.net/aigdata 
```

Comment:
I have some issues w/dotenv (I can go into details more in the next meeting). Go ahead and remove this and the imports etc. You can pretty easily achieve the same effect by using a bash/batch file depending on your os.

Create and .gitignore a `create-env.sh`.
```bash
#!/bin/sh
export MONGODB_URL=mongodb+srv...
```
then just `./create-env.sh` before you run the project.